src/pytorch_lightning/_graveyard/cloud_io.py

```diff
@@ -0,0 +1,23 @@
+import pickle
+import warnings
+from copy import deepcopy
+from typing import Any
+
+
+class RedirectingUnpickler(pickle.Unpickler):
+    def find_class(self, module: str, name: str) -> Any:
+        if module.startswith("pytorch_" + "lightning"):
+            # for the standalone package this won't do anything,
+            # for the unified mirror package it will redirect the imports
```

Comment:
Have you considered cases where the user actually has both installed? Would there be any implications from this change?

Perhaps we should also not run this code if both are available

Reply:
I haven't considered this case so far. But in that case it would depend which package is used to load the checkpoint. I.e when loaded with `lightning.pytorch` I'd argue that this should be executed but if you load with `pytorch_lightning` it should not. And this is also the way it behaves.